docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: "3.8"
+services:
+  front:
+    build:
+      context: ./front
+    ports:
+      - "3000:3000"
+    environment:
+      NEXT_ENV: development
+
+  back:
+    build:
+      context: ./back
+    ports:
+      - "3001:3001"
+    environment:
+      NEXT_ENV: development
+
+  db:
+    image: postgres:16.1
+    restart: always
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_PASSWORD: fake_passwd
```

Comment:
Ok for demo purpose but need to be changed before

Reply:
Postgres password must be declared in an .env.local file in root directory, following the same model as here: "POSTGRES_PASSWORD: fake_passwd"